flux-data-selector.js

```diff
@@ -86,6 +86,7 @@ function login() {
     var credentials = getFluxCredentials();
     if (!credentials) {
         if (!window.location.hash.match(/access_token/)) {
+            localStorage.clear();
```

Comment:
I don't think we want to clear the entire localstorage, maybe just the fluxCredentials property.